src/app/app.component.ts

```diff
@@ -25,7 +25,6 @@ export class AppComponent implements OnInit, OnDestroy {
   languages = [
     { id: "en", name: "English" },
     { id: "hi", name: "Hindi" },
-    { id: "or", name: "Odia" }
```

Comment:
If this is intentional and not temporary then code related to this option should also be removed

Reply:
@gpalsingh - We're in the process of adding Odia as the next feature, but this option has been temporarily disabled.